### PR TITLE
DOC-1777: Changelog edits for 6.3

### PR DESCRIPTION
**Pull request by ltrouton**
Related Ticket: DOC-1777, TINY-8984

Description of Changes:
* Changelog edits courtesy of @brian-forte 

Pre-checks:
* [X] Changelog entry added
* [X] Tests have been added (if applicable)
* [X] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [X] Milestone set
* [X] Docs ticket created (if applicable)

GitHub issues (if applicable):

### CODE REVIEW

**Review by brian-forte — APPROVED** [review]

**Review by tiny-ben-tran — APPROVED** [review]

File: modules/tinymce/CHANGELOG.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - New `expand` function added to `tinymce.selection` which expands the selection around the nearest word. #TINY-9001
 - New `expand` function added to `tinymce.dom.RangeUtils` to return a new range expanded around the nearest word. #TINY-9001
-- New `color_map_background` and `color_map_foreground` option which sets the base colors used in the `backcolor` and `forecolor` toolbar buttons and menu items. #TINY-9184
+- New `color_map_background` and `color_map_foreground` options which set the base colors used in the `backcolor` and `forecolor` toolbar buttons and menu items. #TINY-9184
```
Comment (ephox-mogran) [inline modules/tinymce/CHANGELOG.md:12]: It probably doesn't make it any clearer, but should we put *respectively* in there?

File: modules/tinymce/CHANGELOG.md [hunk 2]
```diff
@@ -9,13 +9,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Added optional `storageKey` property to `colorinput` component and `colorswatch` fancy menu item. #TINY-9184
 - New `addView` function added to `editor.ui.registry` which makes it possible to register custom editor views. #TINY-9210
 - New `ToggleView` command which makes it possible to hide or show registered custom views. #TINY-9210
 - New `color_default_foreground` and `color_default_background` options to set the initial default color for the `forecolor` and `backcolor` toolbar buttons and menu items. #TINY-9183
 - New `getTransparentElements` function added to `tinymce.html.Schema` to return a map object of transparent HTML elements. #TINY-9172
-- Added `ToggleToolbarDrawer` event to subscribe to toolbar's opening and closing. #TINY-9271
+- Added `ToggleToolbarDrawer` event to subscribe to toolbar’s opening and closing. #TINY-9271
```
Comment (ephox-mogran) [inline modules/tinymce/CHANGELOG.md:18]: We actually want smart quotes? But they cause so many problems ...

**Review by ephox-mogran — APPROVED** [review]

File: modules/tinymce/CHANGELOG.md (hunk 2 above)
Comment (brian-forte) [reply modules/tinymce/CHANGELOG.md:18]: > We actually want smart quotes? But they cause so many problems

They also visually distinguish themselves from code-centric quote marks. 'value' is meaningfully different from ‘value’.

Comment (ephox-mogran) [reply modules/tinymce/CHANGELOG.md:18]: Oh they don't cause real problems. They cause "I've copied and pasted this from a docs page into an input field and it doesn't work. A recent case was I copied and pasted something into a query language input and it didn't interpret the smart quotes properly.

File: modules/tinymce/CHANGELOG.md (hunk 1 above)
Comment (TheSpyder) [reply modules/tinymce/CHANGELOG.md:12]: Normally I'd say yes, but because we are describing 4 items (backcolor and forecolor, each with a toolbar button and menu item) I think it would get too wordy. We can't squeeze _respectively_ in the middle, it would have to go at the end, and then what is it respective of?

File: modules/tinymce/CHANGELOG.md (hunk 2 above)
Comment (TheSpyder) [reply modules/tinymce/CHANGELOG.md:18]: 6.3 literally has a feature to convert quotes to smart quotes. I'm not sure where I land on this argument, but we have many non-smart quotes in this changelog already.

**Review by TheSpyder — APPROVED** [review]

File: modules/tinymce/CHANGELOG.md (hunk 2 above)
Comment (ephox-mogran) [reply modules/tinymce/CHANGELOG.md:18]: Sounds more like a bug to me :)

Comment (brian-forte) [reply modules/tinymce/CHANGELOG.md:18]: > but we have many non-smart quotes in this changelog already.

A DOCs ticket in the making, perhaps?

File: modules/tinymce/CHANGELOG.md (hunk 1 above)
Comment (brian-forte) [reply modules/tinymce/CHANGELOG.md:12]: > Normally I'd say yes, but because we are describing 4 items (backcolor and forecolor, each with a toolbar button and menu item) I think it would get too wordy. We can't squeeze respectively in the middle, it would have to go at the end, and then what is it respective of?

FWIW, this was almost literally my thinking when I was doing the copy-edit and trying to make _respectively_ work in the edit that existed before I went with the edit above.
